nemo/utils/exp_manager.py

```diff
@@ -68,6 +68,52 @@ class CheckpointMisconfigurationError(NeMoBaseException):
     """ Raised when a mismatch between trainer.callbacks and exp_manager occurs"""
 
 
+try:
+    import dllogger
+    from dllogger import Verbosity
+
+    class DLLogger(Logger):
```

Comment:
Don't put entire class in try catch. We have pattern of try catched import guard, add a boolean flag, and then at init of class, you check the Boolean and raise error if it's not installed. This is for deferred dynamic import checking

Reply:
Here is an example of an import guard in NeMo: https://github.com/NVIDIA/NeMo/blob/main/nemo/collections/nlp/parts/nlp_overrides.py#L44

Reply:
Done